doc/tutorial/scipy18.rst

```diff
@@ -0,0 +1,11 @@
+*******************
+SciPy 2018 tutorial
+*******************
+
+.. notebook:: pyviz ../../notebooks/scipy18.ipynb
+    :offset: 1
+
+
+-------
+
+`Right click to download this notebook from GitHub. <https://raw.githubusercontent.com/pyviz/pyviz/master/../../notebooks/scipy18.ipynb>`_
```

Comment:
This link appears to be incorrect; it works only if I take out the `../../`:

https://raw.githubusercontent.com/pyviz/pyviz/master/notebooks/scipy18.ipynb